### PR TITLE
Add aws_iam_group_policy_attachment

### PR DESCRIPTION
**Pull request by eliecharra**

### CODE REVIEW

**Review by wbeuil — CHANGES_REQUESTED** [review]
I think it misses the `ListAttachedGroupPolicies` permission.

**Comment by eliecharra** [comment]
> I think it misses the `ListAttachedGroupPolicies` permission.

True, nice catch 🙏🏻

File: docs/providers/aws/authentication.mdx [hunk 1]
```diff
@@ -98,6 +98,7 @@ data "aws_iam_policy_document" "policy" {
       "iam:ListAccessKeys",
       "iam:ListAttachedRolePolicies",
       "iam:ListAttachedUserPolicies",
+      "iam:ListAttachedGroupPolicies",
```
Comment (wbeuil) [inline docs/providers/aws/authentication.mdx:101]: It misses one place for this permission

**Review by wbeuil — CHANGES_REQUESTED** [review]

**Review by wbeuil — APPROVED** [review]
LGTM :)

**Pull request merged**